.github/workflows/continuous-integration.yaml

```diff
@@ -0,0 +1,37 @@
+name: Continuous Integration
+
+on: push
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
```

Comment:
Why do we persist credentials here?

Reply:
fixed with: https://github.com/ridedott/eslint-config/pull/319/commits/267a96360f447175bf63c69e8f81d71b5db5c652